### DISCUSSION TITLE
DateTimePicker

### DISCUSSION THREAD

**Discussion by 15937823** [vmware-clarity/ng-clarity#401] [Ideas]
Hi ! 
Are there plans to add the DateTimePicker component? Or add the function of time selection on the basis of DatePicker?

**Comment by Jinnie** [comment]
cc: @pdaigle

**Comment by Jinnie** [comment]
Moving to Discussions

**Reply by poifir** [reply]
Just want to mention there are ideas from 2017 with lots of comments here: [Date & Time Picker component (new) · Issue #474 · vmware/clarity](https://github.com/vmware/clarity/issues/474) I don't know if @reddolan is still involved in the project? 

Maybe @Jinnie can tell us the current plans for new components?

I noticed there have been major changes in the clarity team (see [Datagrid: add filter for date ranges · Discussion #227 · vmware-clarity/ng-clarity](https://github.com/vmware-clarity/ng-clarity/discussions/227#discussioncomment-4136436) )

So it would be great to get an update to understand the roadmap of the project :) Thanks!

**Reply by Jinnie** [reply]
Our immediate focus is still on accessibility.
Yes, the team is almost entirely changed. The time-picker is on our radar, but without being prioritized yet. Patrick, whom I cc-ed above, is our PM, who's doing some great work on the roadmap now, but we may need some more time before we're ready to share details about our not so immediate future plans.

**Reply by kumar-tadepalli** [reply]
Focusing on accessibility is good.
However, please try to plan to add at least single new component for every major release. Few components like DateTime selector, Range selector with min and max range etc. This way ng-clarity will reduce the gap with other design system libraries. 
We cannot add cds-core for 1 or 2 components.

**Comment by bbogdanov** [comment]
Mentioning the time component in core here

https://storybook.core.clarity.design/?path=/story/forms-time--page

**Comment by map612** [comment]
Expecting a datetime picker

**Comment by goldenmAXE** [comment]
Bump. We are currently relying on an alternative solution for a DateTime picker and it would be nice for this to be included in the Clarity components, particularly for styling consistency.

**Reply by bbogdanov** [reply]
I would advice you to either create your own or rely on an external. A lot of changes happened on a company level and most of the designers inloved with Clarity some devs from the Clarity team where laid off recently.

**Comment by dtsanevmw** [comment]
Hello,

Such component is not part of current backlog and needs so we can't give you any estimates when something like that would come out from Clarity. If such need appears in our backlog we already have solutions prepared that can fast forward the delivery of such component.

**Comment by dtsanevmw** [comment]
Hello, quick update, Date Time picker now appeared in our backlog. It's currently in planning so I can't give estimates when it will be available but just wanted to update that it's part of our backlog and it will be worked on.
